docker-compose.yml

```diff
@@ -0,0 +1,26 @@
+version: '3'
+
+services:
+    scope:
+        restart: always
+        image: scope:1.0
```

Comment:
```suggestion
        build: .
```

Reply:
@bujesse, I think this change is also required to make `docker-compose build` work

Reply:
I made the update, this is much better.